images/win/scripts/Installers/Initialize-VM.ps1

```diff
@@ -90,38 +72,6 @@ Write-Host "Enable long path behavior"
 # See https://docs.microsoft.com/en-us/windows/desktop/fileio/naming-a-file#maximum-path-length-limitation
 Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem' -Name 'LongPathsEnabled' -Value 1
 
-Write-Host "Install chocolatey"
-$chocoExePath = 'C:\ProgramData\Chocolatey\bin'
-
-if ($($env:Path).ToLower().Contains($($chocoExePath).ToLower())) {
-    Write-Host "Chocolatey found in PATH, skipping install..."
-    Exit
-}
-
-# Add to system PATH
-$systemPath = [Environment]::GetEnvironmentVariable('Path', [System.EnvironmentVariableTarget]::Machine)
-$systemPath += ';' + $chocoExePath
-[Environment]::SetEnvironmentVariable("PATH", $systemPath, [System.EnvironmentVariableTarget]::Machine)
-
-# Update local process' path
-$userPath = [Environment]::GetEnvironmentVariable('Path', [System.EnvironmentVariableTarget]::User)
-if ($userPath) {
-    $env:Path = $systemPath + ";" + $userPath
-}
-else {
-    $env:Path = $systemPath
-}
-
-# Run the installer
-Invoke-Expression ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
-
-# Turn off confirmation
-choco feature enable -n allowGlobalConfirmation
-
-# https://github.com/chocolatey/choco/issues/89
-# Remove some of the command aliases, like `cpack` #89
-Remove-Item -Path $env:ChocolateyInstall\bin\cpack.exe -Force
-
 if (Test-IsWin16) {
```

Comment:
Should we move vcredist installation in a separate script?

Reply:
I thought about it but decided to not create separate script for that since this logic is pretty small.
Please let me know if you think it makes sense.

Reply:
Lines from 80 to 87 are also related to the vcredist, so looks like it make sense

Reply:
I think the script Initialize-VM.ps1 is more generic if we move vcredist. And it's the last part of the script which related to 3-rd party tools.

Reply:
Fixed